rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -1,291 +1,3 @@
-        echo "ERROR: The file ${FILE_TO_RESTORE} does not exist and no backup exists. ${RESOLUTION}"
-      fi
-    fi
-  fi
-}
-
-empty() {
-  [ ! -f "$1" ] || [ $(du "$1" | awk '{print $1}') = "0" ]
-}
-
-check_and_fix_inputs() {
-  # if file is absent or empty there have been a problem with update
-  if empty "${CFE_DIR}/inputs/common/1.0/update.cf" || empty "${CFE_DIR}/inputs/failsafe.cf" || empty "${CFE_DIR}/inputs/promises.cf" || ! ${CFE_BIN_DIR}/cf-promises -f failsafe.cf
-  then
-    echo_n "ERROR: There was an error during promises update. Reseting to initial promises and updating..."
-    rm -rf ${CFE_DIR}/inputs/*
-    if [ -d "/opt/rudder/share/initial-promises" ] # rudder package has been properly installed
-    then
-      cp -r /opt/rudder/share/initial-promises/* ${CFE_DIR}/inputs/
-      ${CFE_BIN_DIR}/cf-agent -K -f failsafe.cf && ${CFE_BIN_DIR}/cf-agent
-    else # this is an enterprise only agent
-      if [ -e "${CFE_DIR}/policy_server.dat" ]
-      then
-        ${CFE_BIN_DIR}/cf-agent --bootstrap `cat "${CFE_DIR}/policy_server.dat"`
-      fi
-    fi
-    echo " Done"
-  fi
-
-  # Ensure main promises are not broken, and force a new download if so
-  if ! ${CFE_BIN_DIR}/cf-promises
-  then
-    rm -f ${CFE_DIR}/inputs/rudder_promises_updated
-  fi
-}
-
-# Ensure script is executed by root
-MYUID=`id | cut -d\( -f2 | cut -d\) -f1`
-if [ "${MYUID}" != 'root' ];then echo "You must be root"; exit; fi
-
-# Launch each check with a certain order
-check_and_fix_rudder_uuid
-check_and_fix_specific_rudder_agent_file /var/rudder/cfengine-community/policy_server.dat policy_server.dat "Please configure your Rudder server in this file." "optional"
-
-# Once for rudder's cfengine
-if [ -e "${RUDDER_CFE_DIR}/bin/cf-agent" ]
-then
-  CFE_DIR="${RUDDER_CFE_DIR}"
-  CFE_BIN_DIR="${CFE_DIR}/bin"
-  LAST_UPDATE_FILE="${CFE_DIR}/last_successful_inputs_update"
-  NEED_INIT_FILES="true"
-  check_and_fix_cfengine_processes
-  check_and_fix_cf_lock
-  check_and_fix_inputs
-fi
-
-# Once for enterprise cfengine
-if [ -e "${ENTERPRISE_CFE_DIR}/bin/cf-agent" ]
-then
-  CFE_DIR="${ENTERPRISE_CFE_DIR}"
-  CFE_BIN_DIR="${CFE_DIR}/bin"
-  LAST_UPDATE_FILE="${CFE_DIR}/last_successful_inputs_update"
-  check_and_fix_cfengine_processes
-  check_and_fix_cf_lock
-  check_and_fix_inputs
-fi
-
-# The following files are not present on AIX systems
-if [ "${NEED_INIT_FILES}" = "true" ] && [ "${OS_FAMILY}" != "AIX" ] || [ "${OS_FAMILY}" = "SunOS" ]; then
-  check_and_fix_specific_rudder_agent_file /etc/init.d/rudder-agent rudder-agent.init "Please reinstall the rudder-agent package." "mandatory"
-  check_and_fix_specific_rudder_agent_file /etc/default/rudder-agent rudder-agent.default "Please reinstall the rudder-agent package." "mandatory"
-  check_and_fix_specific_rudder_agent_file /etc/cron.d/rudder-agent rudder-agent.cron "Please reinstall the rudder-agent package." "mandatory"
-fi
+# this file content has been moved
+exec /opt/rudder/share/commands/agent-check
```

Comment:
Same question as in #1003, why not call "rudder agent check" to use the main wrapper?